administration/runtime.md

```diff
@@ -24,97 +24,98 @@ Some of the described commands are executed on the internal database and could b
-| `smarthome:inbox listignored`                | lists all ignored inbox entries    |
-| `smarthome:inbox approve <thingUID> <label>` | creates a thing for an inbox entry |
-| `smarthome:inbox clear`                      | clears all current inbox entries   |
-| `smarthome:inbox ignore <thingUID>`          | ignores an inbox entry permanently |
+| Command                                    | Description                        |
+|--------------------------------------------|------------------------------------|
+| `openhab:inbox`                            | lists all current inbox entries    |
+| `openhab:inbox listignored`                | lists all ignored inbox entries    |
+| `openhab:inbox approve <thingUID> <label>` | creates a thing for an inbox entry |
+| `openhab:inbox clear`                      | clears all current inbox entries   |
+| `openhab:inbox ignore <thingUID>`          | ignores an inbox entry permanently |
 
 ### Things
 
-| Command                  | Description                |
-|--------------------------|----------------------------|
-| `smarthome:things list`  | lists all things           |
-| `smarthome:things clear` | removes all managed things |
+| Command                | Description                |
+|------------------------|----------------------------|
+| `openhab:things list`  | lists all things           |
+| `openhab:things clear` | removes all managed things |
 
 ### Links
 
-| Command                                                   | Description                    |
-|-----------------------------------------------------------|--------------------------------|
-| `smarthome:links list`                                    | lists all links                |
-| `smarthome:links addChannelLink <itemName> <channelUID>`  | links an item with a channel   |
-| `smarthome:links removeChannelLink <itemName> <thingUID>` | unlinks an item with a channel |
-| `smarthome:links clear`                                   | removes all managed links      |
+| Command                                                 | Description                                                              |
+|---------------------------------------------------------|--------------------------------------------------------------------------|
+| `openhab:links list`                                    | lists all links                                                          |
+| `openhab:links link <itemName> <channelUID>`            | links an item with to channel                                            |
+| `openhab:links unlink <itemName> <thingUID>`            | unlinks an item with to channel                                          |
+| `openhab:links clear`                                   | removes all managed links                                                |
+| `openhab:links <list|purge>`                            | lists/purges all links with one missing element (either item or channel) |
```

Comment:
I'm not sure I understand this - is it "either list or purge"? In that case how is the "list" option different to the first line of that table, or is something missing?

Reply:
Yes, either `orphan list` or `orphan purge`. Be aware of the prefix `orphan` 😉.

Reply:
I added the missing piece.